src/ralph/backends/data/clickhouse.py

```diff
@@ -153,69 +169,130 @@ def status(self) -> DataBackendStatus:
     def list(
         self, target: str = None, details: bool = False, new: bool = False
     ) -> Iterator[Union[str, dict]]:
-        """List documents for a given table.
+        """List tables for a given database.
 
         Args:
-            target (str): The table to list documents from.
-            details (bool): Get detailed archive information instead of just ids.
+            target (str): The database name to list tables from.
+            details (bool): Get detailed table information instead of just ids.
             new (bool): Given the history, list only not already fetched archives.
+
+        Yield:
```

Comment:
Is it `Yield` or `Yields`? `Raises` or `Raise`?

Reply:
We should unify docstrings in an imperative mood for the backends, and more generally for Ralph